wallet/substrates/wallet_wire_transceiver.go

```diff
@@ -0,0 +1,51 @@
+package substrates
+
+import (
+	"github.com/bsv-blockchain/go-sdk/wallet"
+	"github.com/bsv-blockchain/go-sdk/wallet/serializer"
+)
+
+// WalletWireTransceiver implements wallet.Interface
+// A way to make remote calls to a wallet over a wallet wire.
+type WalletWireTransceiver struct {
```

Comment:
I think that I'm missing some piece or something, because I don't understand those (`WalletWireTransceiver` & `WalletWireProcessor`) components,
where actually is any transmission via wire?
It looks like it's all happens on the same machine.
Or this is intended for this stage of the implementation, and will change in the future.

Reply:
I think I get it now, the flow which will be handled this way will be like this:
WalletInterface (_implemented by WalletClient_)
```
->  WalletWireTransceiver
->  WalletWire (_implemented by HttpWalletWire_) ~~>
---network---
~~> some server
-> WalletWireProcessor
-> WalletInterface
```

So the HttpWalletWire is missing, and that's why I couldn't understand it.